jekyll/_cci2/pipelines.adoc

```diff
@@ -12,48 +11,63 @@ contentTags:
+NOTE: The pipeline and trigger features available to you depend on how your code is integrated and where it is stored. For a feature availability overview, see the xref:version-control-system-integration-overview#[VCS integration overview] page.
+
+NOTE: **Using CircleCI server?** The Pipeline and Trigger pages under Project Settings and pipeline and trigger add/edit functionality is **not** available.
+
+Set up pipelines to define the code to checkout and the config to use for your project CI/CD..
+
+Set up triggers to define when and how a pipeline is triggered for a project. Triggers may be from external services (custom webhooks), from commits or tags on a version control system, or set up to run on a schedule.
 
 === Pipelines
 
-Pipelines define units of work that can be run within your projects.
-Each pipeline is defined by two key characteristics:
+A pipeline defines a unit of work that can be run within your projects. Each pipeline is defined by two key characteristics:
+
+* A config source, which is a configuration file stored in a repository. The default config source location is `.circleci/config.yml` within the same repositry as your code.
++
+TIP: For GitHub App and Bitbucket Data Center pipelines you can choose any file path (with extension `*.yml`) in any repository.
+
+* A checkout source, which determines which repository will be checked out when running a xref:configuration-reference#checkout[checkout step].
++
+TIP: This model allows your CircleCI config to be stored in a different repository from your code. This is currently supported for GitHub App and Bitbucket Data Center pipelines.
 
-* A configuration file, which is stored in a repository referred to as config source, at a path of your choosing. The default config source location is `.circleci/config.yml`, but any file path with extension `*.yml` can be used.
-* A checkout source, which determines which repository will be checked out when running a xref:configuration-reference#checkout[checkout step]. This model allows your CircleCI config to be stored in a different repository from your code.
+A project can have one or more pipelines, and each pipeline can be associated with zero or more triggers.
 
-A project can have one or more pipelines, and each pipeline can be associated with one or more triggers.
+When you create a project in CircleCI and follow the in-app project creation flow to connect it to a repository, a first pipeline is set up for you automatically. The config source is either a file you commit yourself or one generated by CircleCI based on the code in your project. You can identify the type (GitHub App, OAuth, custom webhook, scheduled) of a pipeline using the badges in the Config Source column on the menu:Project Settings[Pipelines] page.
```

Comment:
> You can identify the type (GitHub App, OAuth, custom webhook, scheduled) of a pipeline using the badges in the Config Source column on the menu:Project Settings[Pipelines] page.
I think this refers to type of triggers rather than pipelines? 

```suggestion
When you create a project in CircleCI and follow the in-app project creation flow to connect it to a repository, a first pipeline is set up for you automatically. The config file used for your first pipeline is either a file you commit yourself or one generated by CircleCI based on the code in your project.
```